### ISSUE TITLE
Category Grid Block

### ISSUE THREAD

**Issue by nerrad** [woocommerce/woocommerce-gutenberg-products-block#2380]
This block introduces a way for merchants to insert a grid of categories (including images for those categories) linked to the product archive page for those categories.

**P2:** (pbIy4N-8I-p2)

**Comment by mikejolley** [comment]
I'm going to ignore this one until designs are ready; please ping me when there is something to look at.  @nerrad 

@garymurray Whilst designing this, you may want to look at the existing Store API response for categories to get an idea of what data is already available to you. 

```curl
GET wc/store/products/categories
```

This is an example response:

```json
[
{
    "id": 16,
    "name": "Clothing",
    "slug": "clothing",
    "description": "",
    "parent": 0,
    "count": 14,
    "image": {
      "id": 56,
      "src": "https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1.jpg",
      "thumbnail": "https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-324x324.jpg",
      "srcset": "https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1.jpg 800w, https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-324x324.jpg 324w, https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-100x100.jpg 100w, https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-416x415.jpg 416w, https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-300x300.jpg 300w, https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-150x150.jpg 150w, https:\/\/local.wordpress.test\/wp-content\/uploads\/2020\/03\/logo-1-768x767.jpg 768w",
      "sizes": "(max-width: 800px) 100vw, 800px",
      "name": "logo-1.jpg",
      "alt": ""
    },
    "review_count": 0,
    "permalink": "https:\/\/local.wordpress.test\/product-category\/clothing\/"
  }
]
```

**Comment by nerrad** [comment]
cc @LevinMedia as well. I'm assuming you'll be doing the designs?

**Comment by maikunari** [comment]
Is this block still a possibility?

**Comment by nerrad** [comment]
Yes, it's still on our radar as something we want to build, but I do not have any ETA for when it'll happen.

**Comment by phil-cariad** [comment]
Is there any news on this block becoming a realilty? it is a gaping hole in the WooCommerce blocks library for site builders currently. The only option for displaying categories on any page, is: 

a.) using a WooCommerce shortcode
b.) using the featured category block which only displays 1 way with no option to change it. 

There aren't even any 3rd party solutions that I can find and I have searched for a long time. 

I would love to see this come to WooCommerce soon ideally. Building an ecommerce store without being able to scatter categories is really tough. It's one of the things I dread in WooCommerce builds right now.

**Comment by asafm7** [comment]
Any updates by any chance?

I'm not a designer, but can't the design be similar to the product grid?

It would be great to be able to place such a block on a category page to display only the direct subcategories of the category, before the products. It used to be possible with the old customizer
